index.html

```diff
@@ -16,29 +16,58 @@
   <header>
     <navbar class="nav-dsk">
       <a href="index.html"><img src="images/logo.png" alt="logo" class="logo"></a>
+      <div class="dark-nav">
+        <ul>
+          <li><a href="#"><i class="fa-brands fa-facebook-f"></i></a></li>
+          <li><a href="#"><i class="fa-brands fa-twitter"></i></a></li>
+          <li><a href="#">English</a></li>
+          <li><a href="#">My Page</a></li>
+          <li><a href="#">Logout</a></li>
+        </ul>
+      </div>
       <ul>
-        <li><a href="index.html">Home</a></li>
         <li><a href="about.html">About</a></li>
+        <li><a href="about.html">Program</a></li>
+        <li><a href="about.html">Join</a></li>
+        <li><a href="about.html">Sponsors</a></li>
+        <li><a href="about.html">News</a></li>
         <li class="menu-highlight"><a href="#">Stoa Campaign</a></li>
       </ul>
     </navbar>
```

Comment:
- **[OPTIONAL]** I highly recommend using `nav` tag instead, since there is no such tag such as `navbar` in html.